test/evergreen.yml

```diff
@@ -2965,6 +2965,73 @@ tasks:
       - func: "code coverage publish main page"
       - func: "code coverage publish report"
 
+  - name: coverage-report-per-test
+    # This task will measure code coverage across a range of tests, including, but not limited to,
+    # the Catch2-based unit tests. The tests are run in parallel, and data is captured seperately per test.
+    commands:
+      - func: "get project"
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger"
+          shell: bash
+          script: |
+            set -o errexit
+            set -o verbose
+            ${PREPARE_TEST_ENV}
```

Comment:
Could we place this script into a bash script? It seems quite complex as is.

Reply:
Great idea - now done!